load_testing/scenarios.py

```diff
@@ -36,3 +37,26 @@ def test(url):
         test('/data/Discussion/{disc_id}/')
         test('/data/Discussion/{disc_id}/idea_links')
         test('/api/v1/discussion/{disc_id}/posts?{posts}&order=reverse_chronological&view=default')
+
+    def notification_subscriptions(self):
```

Comment:
Note to developers, the conditions to run this test vary widely from the typical use-case of running `pytest`.